src/Cache/Interceptors/AgentInterceptor.php

```diff
@@ -10,10 +10,12 @@ class AgentInterceptor extends Interceptor
     private bool $isFirstRequest = true;
     private string $agent;
     private string $runtimeVersion;
+    private string $sdkVersion = "1.10.0"; // x-release-please-version
```

Comment:
Do we would need to manually update the sdkVersion whenever we bump the release-please version?

Reply:
I believe release-please knows to update that line based on the `// x-release-please-version`

Reply:
Interesting, thanks for letting me know!

Reply:
Yep, the `extra-files` option in the gh action tells rp to find and update a version string in this file.